cli/command/service/create.go

```diff
@@ -106,6 +106,11 @@ func runCreate(dockerCli *command.DockerCli, flags *pflag.FlagSet, opts *service
 		createOpts.EncodedRegistryAuth = encodedAuth
 	}
 
+	// query registry if flag disabling it was not set
+	if !opts.noResolveImage {
```

Comment:
@thaJeztah @aaronlehmann is this right? In my local testing, the option doesn't seem to be set as I expect. But other flags are also used this way in `service/create.go`, so not sure what I'm missing.

The default should be `noResolveImage = false`, and if `--no-resolve-image` is passed, then `noResolveImage` should be set to `true`.